ts/client/src/client.ts

```diff
@@ -2265,6 +2265,52 @@ export class MangoClient {
     return await this.sendAndConfirmTransactionForGroup(group, [ix]);
   }
 
+  public async perpCloseAll(
+    group: Group,
+    mangoAccount: MangoAccount,
+    slippage = 0.01, // 1%, 100bps
+  ): Promise<TransactionSignature> {
+    if (mangoAccount.perpActive().length > 0) {
+      throw new Error(`No perp positions found.`);
+    }
+
+    if (mangoAccount.perpActive().length > 8) {
```

Comment:
optionally save a lot of compute by using a health region (at least leave this as a suggestion here)